packages/aws-cdk/lib/api/hotswap/appsync-mapping-templates.ts

```diff
@@ -1,29 +1,30 @@
-import { ChangeHotswapResult, classifyChanges, HotswappableChangeCandidate, lowerCaseFirstCharacter, reportNonHotswappableChange, transformObjectKeys } from './common';
+import { GetSchemaCreationStatusRequest, GetSchemaCreationStatusResponse } from 'aws-sdk/clients/appsync';
+import { ChangeHotswapResult, classifyChanges, HotswappableChangeCandidate, lowerCaseFirstCharacter, transformObjectKeys } from './common';
 import { ISDK } from '../aws-auth';
+
 import { EvaluateCloudFormationTemplate } from '../evaluate-cloudformation-template';
 
 export async function isHotswappableAppSyncChange(
   logicalId: string, change: HotswappableChangeCandidate, evaluateCfnTemplate: EvaluateCloudFormationTemplate,
 ): Promise<ChangeHotswapResult> {
   const isResolver = change.newValue.Type === 'AWS::AppSync::Resolver';
   const isFunction = change.newValue.Type === 'AWS::AppSync::FunctionConfiguration';
+  const isGraphQLSchema = change.newValue.Type === 'AWS::AppSync::GraphQLSchema';
 
-  if (!isResolver && !isFunction) {
+  if (!isResolver && !isFunction && !isGraphQLSchema) {
     return [];
   }
 
   const ret: ChangeHotswapResult = [];
-  if (isResolver && change.newValue.Properties?.Kind === 'PIPELINE') {
-    reportNonHotswappableChange(
-      ret,
-      change,
-      undefined,
-      'Pipeline resolvers cannot be hotswapped since they reference the FunctionId of the underlying functions, which cannot be resolved',
-    );
-    return ret;
-  }
```

Comment:
is this no longer true?

Reply:
Yes, now that we can resolve the functionIds https://github.com/aws/aws-cdk/pull/27197/files#diff-ff9724092d7f1e6bf3141e51ddb00ecbc2148e16f141b0bd2074d6edc60c16b6R394-R398